backend/src/main/java/com/google/sps/data/Time.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import java.util.Calendar;
+
+/** Class to convert Time */
+public final class Time {
+
+  private static Date date;
```

Comment:
why static?

Reply:
I got rid of this static, made the getTimeMillis() method static, and changed the references in TimeTest.java

Reply:
Edit: I changed it back to being static for it to be referenced easily in the Test. For now, this method is only used in the tests, but I also wanted it to be used when setting timeAvailableUntil later.